nixos/modules/services/networking/ntp/ntpd-rs.nix

```diff
@@ -74,13 +74,13 @@ in
       };
     };
 
-    systemd.services.ntp-rs-metrics = lib.mkIf cfg.metrics.enable {
+    systemd.services.ntpd-rs-metrics = lib.mkIf cfg.metrics.enable {
       wantedBy = [ "multi-user.target" ];
       serviceConfig = {
         User = "";
         Group = "";
         DynamicUser = true;
-        ExecStart = [ "" "${lib.makeBinPath [ cfg.package ]}/bin/ntp-metrics-exporter --config=${configFile}" ];
+        ExecStart = [ "" "${lib.makeBinPath [ cfg.package ]}/ntp-metrics-exporter --config=${configFile}" ];
```

Comment:
This won't work since there is no `meta.mainProgram`.

Reply:
> Also I want to prioritize high quality reviews with explanations over quantity. I think I got already tied less to attention to details (which was a problem for people before).

Reply:
> This won't work since there is no `meta.mainProgram`.

We could have added that instead. Using makeBinPath feels a bit hacky here and lib.getExe is meant for this exact usecase.

---

I am not sure what you are trying to say with that globin. It feels a bit passive aggressive to me.

Reply:
> We could have added that instead. Using makeBinPath feels a bit hacky here and lib.getExe is meant for this exact usecase.

Please review more carefully. This is the metrics exporter service binary of `ntpd-rs` which certainly won't ever be the `mainProgram` of the package.